katalog/x509-exporter/daemonset/x509-certificate-exporter-control-plane/kustomization.yaml

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2021 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: monitoring
+
+nameSuffix: -control-plane
+
+patchesStrategicMerge:
+- |-
+  apiVersion: apps/v1
+  kind: DaemonSet
+  metadata:
+    name: x509-certificate-exporter
+    namespace: monitoring
+  spec:
+    template:
+      spec:
+        affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: node-role.kubernetes.io/control-plane
+                    operator: Exists
+                - matchExpressions:
+                  - key: node-role.kubernetes.io/master
+                    operator: Exists
+        containers:
+          - name: x509-certificate-exporter
+            volumeMounts:
+              - name: etc-etcd-pki
+                mountPath: /mnt/watch//etc/etcd/pki
```

Comment:
is the double slash in the path ok?

Reply:
Yes, UNIX path normalization considers `//` as `/`.